src/server/game/Entities/Object/Object.cpp

```diff
@@ -1141,6 +1143,11 @@ bool WorldObject::_IsWithinDist(WorldObject const* obj, float dist2compare, bool
     return distsq < maxdist * maxdist;
 }
 
+bool WorldObject::_IsWithinDist(WorldObject const* obj, float dist2compare, bool is3D) const
+{
+    return _IsWithinDist(obj, dist2compare, is3D, true, true);
```

Comment:
WorldObject::_IsWithinDist(WorldObject const* obj, float dist2compare, bool is3D) const is overriden in GameObject. I didn't want to rewrite that method because I'm not sure what it does exactly/ how it does it.

So I created a separate method in order to not affect the rest of the code. Maybe not the best approach..